Create what-if-it-works.yaml

pdpinch opened the pull request:
Adding podcast from MIT Energy Initiative

### What are the relevant tickets?
<!--- If it fixes an open issue, please link to the issue here. -->
<!--- Closes # --->
<!--- Fixes # --->
 
In response to Appzi feedback: https://mitodl.slack.com/archives/C08NNLP3R5X/p1753970520721999

### Description (What does it do?)
<!--- Describe your changes in detail -->

Adds MITEI's "What if it Works" podcast 

### Todo 

I need to double-check the topics

pdpinch committed:
Create what-if-it-works.yaml

Adding podcast from MIT Energy Initiative

pdpinch committed:
update topics 

There is no Engineering topic in Apple's list. But there is Technology.

pdpinch committed:
Update what-if-it-works.yaml

responding to Gemini's comments. 

Note, Google podcasts has been shutdown.

pdpinch committed:
Rename what-if-it-works.yaml to podcasts/what-if-it-works.yaml

moving it to the correct directory

pdpinch commented:
/gemini review

abeglova commented on podcasts/what-if-it-works.yaml:
```diff
@@ -0,0 +1,6 @@
+---
+rss_url: https://feeds.megaphone.fm/whatifitworks
+topics: Science,Technology 
```
`Science` and `Technology` are not valid topics since they are so broad.  Here is the list of allowed topics
https://api.learn.mit.edu/api/v1/topics/

abeglova reviewed (changes requested):
The topics are not valid

pdpinch committed:
Update podcast topics to 'Energy, Climate and Energy Policy'

pdpinch commented on podcasts/what-if-it-works.yaml:
```diff
@@ -0,0 +1,6 @@
+topics: Science,Technology 
+offered_by: MIT Energy Initiative
+website: https://energy.mit.edu/podcasts/
+apple_podcasts_url: https://podcasts.apple.com/us/podcast/what-if-it-works/id1772326078
```
Google's moved it's podcast service to YouTube. I guess this will work? https://www.youtube.com/playlist?list=PLPhedGFHe8CnuwbG6eZVy4_kgfKC4goFX

pdpinch committed:
Add Google Podcasts URL to what-if-it-works.yaml

Google podcasts site has been retired in favor of YouTube. I'm not sure if this URL will work with Open and Learn as is.

pdpinch commented:
@abeglova I updated the topics. Can you take another look?

abeglova reviewed (approved):
lgtm
